src/OpenLoco/src/Entities/EntityManager.h

```diff
@@ -51,11 +48,6 @@ namespace OpenLoco::EntityManager
     EntityBase* createEntityVehicle();
```

Comment:
Should these be moved to their respective managers?

Reply:
Ideally yes but I wanted to avoid makin this PR too big, entity types should annotate their type and list type, I planned to work on that in the next update.